lib/interfaces/common.js

```diff
@@ -130,12 +133,14 @@ module.exports = function(suites, context, mocha) {
           if (mocha.options.forbidOnly && shouldBeTested(suite)) {
```

Comment:
Is this part really necessary? When `mocha.options.forbidOnly` you already have thrown above.

Reply:
You are right, thanks! I removed the condition.